opm/simulators/utils/ParallelRestart.cpp

```diff
@@ -409,6 +409,27 @@ struct Packing<data::Segment>
     }
 };
 
+template <size_t Size>
```

Comment:
Is there a reason for using `size_t` instead of `std::size_t` here?

Reply:
no reason, just a slip. fixed now.